kratos/includes/condition.h

```diff
@@ -1056,7 +1056,7 @@ class Condition : public GeometricalObject
     }
 
     virtual void SetValuesOnIntegrationPoints(const Variable<array_1d<double, 3 > >& rVariable,
-                         std::vector<array_1d<double, 3 > > rValues,
+                         std::vector<array_1d<double, 3 > >& rValues,
```

Comment:
I think so... I think it will be the same as the `const ProcessInfo`.

Reply:
As reminder this is the same in the Element

Reply:
We can do that pretty fast by just adapting the affected elements/conditions in each applications accordingly. But we have no experience with the fluid-related applications so it may bother the authors. How about making a poll if the authors accept touching their apps?

Reply:
Based on the experiences with #7381 I would do this change in one sweep. Having both versions coexist with the only difference of one argument being `const` is a big pain
I checked, there are 358 usages in the code, to me this is doable in one PR